django/thunderstore/social/api/experimental/views/current_user.py

```diff
@@ -96,5 +98,8 @@ def get_subscription_status(user: Optional[UserType]) -> SubscriptionStatus:
     if not user:
         return {"expires": None}
 
-    # TODO: return expiration date if user has active subscription.
+    now = datetime.datetime.now()
```

Comment:
use Django's `timezone.now()` as you generally always should to ensure timezone information is properly included.